src/daemon/bsd/plugins/meson.build

```diff
@@ -0,0 +1,185 @@
+all_plugins = [
+#    ['a11y-settings', 'A11ySettings', 'GNOME accessibility'],
+#    ['color', 'Color', 'GNOME color management'],
+#    ['datetime', 'Datetime', 'GNOME date & time'],
+     ['power', 'Power', 'GNOME power management'],
+#    ['housekeeping', 'Housekeeping', 'GNOME maintenance of expirable data'],
+#    ['keyboard', 'Keyboard', 'GNOME keyboard configuration'],
+    ['media-keys', 'MediaKeys', 'GNOME keyboard shortcuts'],
+#    ['screensaver-proxy', 'ScreensaverProxy', 'GNOME FreeDesktop screensaver'],
+#    ['sharing', 'Sharing', 'GNOME file sharing'],
+#    ['sound', 'Sound', 'GNOME sound sample caching'],
+#    ['usb-protection', 'UsbProtection', 'GNOME USB protection'],
+#    ['xsettings', 'XSettings', 'GNOME XSettings'],
+#    ['smartcard', 'Smartcard', 'GNOME smartcard'],
+    ['wacom', 'Wacom', 'GNOME Wacom tablet support'],
+#    ['print-notifications', 'PrintNotifications', 'GNOME printer notifications'],
+#    ['rfkill', 'Rfkill', 'GNOME RFKill support'],
+#    ['wwan', 'Wwan', 'GNOME WWan support'],
+]
+
+disabled_plugins = []
+
+if not enable_smartcard
+    disabled_plugins += ['smartcard']
+endif
+
+if not enable_usb_protection
+    disabled_plugins += ['usb-protection']
+endif
+
+if not enable_wacom
+    disabled_plugins += ['wacom']
+endif
+
+if not enable_cups
+    disabled_plugins += ['cups']
+endif
+
+if not enable_rfkill
+    disabled_plugins += ['rfkill']
+endif
+
+if not enable_wwan
+    disabled_plugins += ['wwan']
+endif
+
+if not enable_colord
+  disabled_plugins += ['color']
+endif
+
+if not enable_cups
+  disabled_plugins += ['print-notifications']
+endif
+
+# Specify futher required units, 'before' or 'after' may be specified if ordering is needed
+plugin_gate_units = {
+    'xsettings': [
+      # Both after/before. after for stopping reliably, before for synchronisation
+      ['gnome-session-x11-services.target', 'after'],
+      ['gnome-session-x11-services-ready.target', 'before'],
```

Comment:
Note to self: Need to double check if these have been nuked in GNOME Session or Fedora.

Reply:
These still exist in F43 gnome-session at least as of 49.rc